Assets/Scripts/Game/Tasks/GameTask.cs

```diff
@@ -50,6 +50,8 @@ public abstract class GameTask : MonoBehaviour
         /// </summary>
         private readonly List<GameObject> m_LinkedGameObjects = new List<GameObject>();
         
+        public TaskSpawnPoint spawnPoint;
```

Comment:
```suggestion
```
This variable is set and updated in `TaskSpawnPoint.cs` but never used by any script.